benchexec/tools/metaval++.py

```diff
@@ -0,0 +1,97 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import logging
+
+import benchexec.result
+from benchexec.tools.sv_benchmarks_util import ILP32, LP64, get_data_model_from_task
+from benchexec.tools.template import BaseTool2
+
+
+class Tool(BaseTool2):
+    """
+    Tool info for Deductive Validator.
+    """
+
+    REQUIRED_PATHS = ["lib", "src", "metaval++.py"]
```

Comment:
```suggestion
    REQUIRED_PATHS = ["lib", "src"]
```
The executable is added to the program files anyway by `program_files()`.

Reply:
Thanks a lot for pointing this out!

I was not aware of it, has been fixed now